sherlockml/clients/server.py

```diff
@@ -92,7 +83,10 @@ class ServerSchema(Schema):
     )
     server_size = fields.Nested(ServerSizeSchema, data_key="instanceSize")
     created_at = fields.DateTime(data_key="createdAt", required=True)
-    status = EnumField(ServerStatus, by_value=True, required=True)
+    status = fields.String(
+        required=True,
+        validate=validate.OneOf(["creating", "running", "error", "destroyed"]),
```

Comment:
I have a mild preference for moving this list out into a module constant for parity with `sherlockml/clients/job.py`.